### DISCUSSION TITLE
Regex manager with new Patch version

### DISCUSSION THREAD

**Discussion by illay1994** [renovatebot/renovate#29340] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure

### Please tell us more about your question or problem

I try to add new custom manager for Renovate

```
 {
            "customType": "regex",
            "fileMatch": [
              "Workspace.swift$"
            ],
            "matchStrings": [
                "generationOptions: \\.options\\(lastXcodeUpgradeCheck: Version\\( (?<majorV>\\d*), (?<minorV>\\d*), (?<patchV>\\d*) \\)\\)"
            ],
            "currentValueTemplate": "{{majorV}}.{{minorV}}{{#unless (equals patchV '0')}}.{{patchV}}{{/unless}}",
            "autoReplaceStringTemplate": "generationOptions: .options(lastXcodeUpgradeCheck: Version( {{{newMajor}}}, {{{newMinor}}}, 0 ))",
            "depNameTemplate": "xcodereleases",
            "datasourceTemplate": "custom.xcodereleases",
            "versioningTemplate": "semver-coerced"  
          },
```

how i can add new {{{newPatch}}} version if new patch version available?

I try to use next: {{#if isPatch }}{{{unknownValue}}}{{else}}0{{/if}}
### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment]
use `newMajor`, `newMinor` and `newPatch`

https://docs.renovatebot.com/templates/
